paseos/attitude/disturbance_torques_utils.py

```diff
@@ -0,0 +1,262 @@
+                * np.exp(-((molecular_speed_ratio_t * np.sin(alpha_scalar)) ** 2))
+                + np.sin(alpha_scalar)
+                * (
+                    1
+                    + 2 * molecular_speed_ratio_t**2
+                    + (1 - accommodation_parameter)
+                    * (1 - 2 * molecular_speed_ratio_t**2 * np.cos(2 * alpha_scalar))
+                )
+                * np.math.erf(molecular_speed_ratio_t * np.sin(alpha_scalar))
+                + accommodation_parameter
+                * np.sqrt(np.pi)
+                * molecular_speed_ratio_t**2
+                / molecular_speed_ratio_r
+                * np.sin(alpha_scalar) ** 2
+            )
+        )
+
+        C_l = (
+            np.cos(alpha[k])
+            / molecular_speed_ratio_t**2
+            * (
+                2
+                / np.sqrt(np.pi)
+                * (2 - 2 * accommodation_parameter)
+                * molecular_speed_ratio_t
+                * np.sin(alpha_scalar)
+                * np.exp(-((molecular_speed_ratio_t * np.sin(alpha_scalar)) ** 2))
+                + (
+                    2
+                    * (2 - 2 * accommodation_parameter)
+                    * (molecular_speed_ratio_t * np.sin(alpha_scalar)) ** 2
+                    + (2 - accommodation_parameter)
+                )
+                * np.math.erf(molecular_speed_ratio_t * np.sin(alpha_scalar))
+                + accommodation_parameter
+                * np.sqrt(np.pi)
+                * molecular_speed_ratio_t**2
+                / molecular_speed_ratio_r
+                * np.sin(alpha_scalar)
+            )
+        )
+
+        # Drag force on the plate [k]. Direction along the velocity vector.
+        force_drag[k] = (
+            -0.5 * density * C_d * area_faces_airflow[k] * np.linalg.norm(v) ** 2 * unit_v_rpy
+        )
+        # Lift force on the plate [k]. Direction along the (v x n) x v direction, lift vector defined to be in that
+        # direction. Intermediate step to get v x n.
+        v_x_n_vector = np.cross(unit_v_rpy, normals_faces_with_airflow[k])
+        not_norm_lift_vector = np.cross(v_x_n_vector, unit_v_rpy)
+        lift_vector = not_norm_lift_vector / np.linalg.norm(not_norm_lift_vector)
+        force_lift[k] = (
+            -0.5 * density * C_l * area_faces_airflow[k] * np.linalg.norm(v) ** 2 * lift_vector
+        )
+
+        # Torque calculated as the product between the distance of the centroid from the geometric center of the
+        # satellite and the sum of the forces.
+        torque_aero[k] = np.cross(
+            centroids_faces_airflow[k] - mesh.centroid,
+            (np.array(force_drag[k]) + np.array(force_lift[k])),
+        )
+
+    #  Compute aero torque in the timestep as the vector sum of all the torques
+    torque_aero = np.array(torque_aero)
+    T_rpy = torque_aero.sum(axis=0)
+    T = transformation_matrix_rpy_body @ T_rpy
+    T[np.isnan(T)] = 0  # substitutes 0 to any NaN value
```

Comment:
? why are there NaN values in the first place? this sounds like there is a bug somewhere